__test__/item.test.js

```diff
@@ -0,0 +1,17 @@
+var Shop = require('../src/shop');
+var Item = require('../src/item');
+
+describe("item test", function() {
+	it('Item name does not change after update', () => {
+		const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
+	    const items = gildedRose.updateQuality();
+	    expect(items[0].name).toBe("foo");
+    });
+
+  it('Item count does not change after update', () => {
+	const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
```

Comment:
You are creating the same shop twice for two different test, could we refactor this to remove redundancy?